client/app/scripts/superdesk-content-filters/content-filters.js

```diff
@@ -425,7 +436,10 @@
 
         var fetchContentFilters = function() {
             contentFilters.getAllContentFilters().then(function(_filters) {
-                $scope.contentFilters = _filters;
+                $scope.contentFilters = _.sortBy(_filters, function(filter) {
+                    return filter.name.toLowerCase();
+                });
```

Comment:
Even I thought the same. Since it depends on the resource properties which are controlled by server I thought extracting it to common function may not be relevant.